pages/content/amp-dev/documentation/courses/beginning-course/thinking-in-components@pt_BR.md

```diff
@@ -0,0 +1,265 @@
+
+{{ image('/static/img/courses/beginner/image4.png', 297, 380,  align='center half', caption='Lista suspensa de dispositivos no Chrome') }}
+[/tip]
+
+## Exercício 4: Incorporação de Vídeo
+
+A seguir, vamos incorporar um vídeo do YouTube em nosso documento. Nossa equipe de marketing da Chico’s Bikes lançou [este](https://www.youtube.com/watch?v=BlpMQ7fMCzA) vídeo de uma de nossas bicicletas queijos sendo construídas.
+
+Use a documentação do [`<amp-youtube>`](../../../documentation/components/reference/amp-youtube.md) para incorporar este vídeo do YouTube no componente `<amp-img>` com as seguintes configurações:
+
+- Defina o id do vídeo para `BlpMQ7fMCzA`.
+
+- Faça o layout do vídeo `responsive`.
+
+- **Nota**: Não se esqueça de adicionar o script ao `<head>`.
+
+Diretrizes de estilo recomendadas:
+
+- Defina o elemento `width` como` 480` e a `height` como` 270`.
+
+[tip type="read-on"]
+**Dica**: A documentação contém exemplos de como usar o componente `<amp-youtube>`. Para este exercício, é suficiente copiar um desses exemplos e adaptá-lo aos requisitos acima.[/tip]
+
+Depois de fazer as alterações, olhe para sua página. Agora você deve ver o vídeo do YouTube:
+
+{{ image('/static/img/courses/beginner/image18.png', 311, 550,  align='center third', caption='Imagem do vídeo do YouTube na página') }}
+
+### Solução
+
+[sourcecode:html]
+{% raw %}<amp-youtube data-videoid="BlpMQ7fMCzA" layout="responsive" width="480" height="270"></amp-youtube>
+{% endraw %}[/sourcecode]
+
+Lembre-se de incluir o script `<amp-youtube>` no `<head>`:
+
+```
+<script async custom-element="amp-youtube" src="https://cdn.ampproject.org/v0/amp-youtube-0.1.js"></script>
+```
+
+[tip type="note"]
+**Nota**: AMP também inclui suporte para outros players de vídeo. Confira "[Integrando Vídeos em AMP e Visão Geral](https://ampbyexample.com/advanced/integrating_videos_in_amp_an_overview/)" em AMP por exemplos.[/tip]
+[/filter]
+[filter level="advanced"]
+Há apenas um erro de validação restante em nosso site, e vamos corrigi-lo usando um **componente** `<amp-img>` em vez de uma tag `<img>`.
+
+## Oque são Web Components?
+
+Os componentes são blocos de construção para a web. Eles combinam estrutura (HTML), estilo (CSS) e comportamento (JavaScript) com uma interface simples. Os componentes têm:
+
+- Um nome (por exemplo, `<amp-img>`) usado como o nome da tag para identificar o componente.
+
+- Atributos personalizados que alteram o comportamento, estilo ou conteúdo de um componente (como `width`,` height`, `src` e`attribution`).
+
+- Eventos que podem capturar entradas do usuário para o componente (atributo ʻon`).
```

Comment:
apostrophe and backticks in the word on probably should be backticks and backticks or apostrophe and apostrophe.

Reply:
Done